src/provider.ts

```diff
@@ -74,10 +83,54 @@ export class Provider {
     }
 
     // submit a transaction to be signed by the wallet to the websocket
-    async submitTransaction(payload: { commands: any[]; disclosedContracts: any[] }): Promise<any> {
+    async submitTransaction(payload: TransactionPayload): Promise<any> {
         return this.sendRequest(MessageType.RUN_TRANSACTION, payload);
     }
 
+    async transfer(
+      recipient: string,
+      amount: string | number,
+      options?: TransferOptions,
```

Comment:
at API level, it make more sense for { isntrument_admin, instrument id} to become an argument on its own.  Cuz these are always required, we just auto falbacll to Amulet but they are requirement at API level.

so i would change this to

```
            receiver,
            'amount',
            {
                instrumentId: 'Amulet',
                instrumentAdmin: instrumentAdminPartyId,
            },
            { memo: , executedBefore: etc }, # options here
```